src/components/start-project-overlay.tsx

```
@@ -0,0 +1,104 @@
+import { Input } from "@/components/ui/input"
+import { Label } from "@/components/ui/label"
+import { Textarea } from "@/components/ui/textarea"
+import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from "@/components/ui/select"
+import confetti from "canvas-confetti"
+
+interface StartProjectOverlayProps {
+  isOpen: boolean
+  onClose: () => void
+}
+
+const categories = ["Education", "Community", "Technology", "Environment", "Arts & Culture", "Wellness"]
```

ESLint

[error] Strings must use singlequote.

(style/quotes)